src/PickerTop.js

```diff
@@ -13,10 +13,12 @@ import { pure, skinnable } from './utils';
   nextDate: t.maybe(t.Function),
   previousDate: t.maybe(t.Function),
   value: t.union([t.String, t.Number]),
-  weekDays: t.maybe(t.ReactChildren)
+  weekDays: t.maybe(t.ReactChildren),
+  prevIconClassName: t.maybe(t.String),

```

Comment:
`prevIconClassName` and `nextIconClassName` should be required as it's `DatePicker`'s duty to add a default in case they're missing